lib/tracking.js

```diff
@@ -1,7 +1,29 @@
-function setup(client) {
+var MIN_HOVER_TIME = 200;
+
+function setupMouseHover(client) {
+  var overStart = Date.now();
+
+  window.addEventListener('mouseover', function() {
+    overStart = Date.now();
+  });
+
+  window.addEventListener('mouseout', function() {
+    var overFor = Date.now() - overStart;
+    if(overFor >= MIN_HOVER_TIME) {
+      client.invoke('track', { type: 'hover',  hoverLength: overFor });
+    }
+  });
+}
```

Comment:
If you changed lines 10-16 like so:

``` javascript
function hoverHandler() {
  var overFor = Date.now() - overStart;
  if(overFor >= MIN_HOVER_TIME) {
    client.invoke('track', { type: 'hover',  hoverLength: overFor });
  }
}

window.addEventListener('mouseout', hoverHandler);
```

Then you could refer to `tracking.hoverHandler` directly in the tests without needing to use `findEvent` to look for it on the `eventListener`.